catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/reader/TestGetRecordsMessageBodyReader.java

```diff
@@ -151,53 +152,23 @@ public void testFullThread() throws Exception {
         List<Metacard> metacards = cswRecords.getCswRecords();
 
         assertThat(metacards, not(nullValue()));
-        assertThat(metacards.size(), equalTo(3));
+        assertThat(metacards.size(), equalTo(1));
 
         // verify first metacard's values
         Metacard mc = metacards.get(0);
-        Map<String, Object> expectedValues =
-                getExpectedMap("{8C1F6297-EC96-4302-A01E-14988C9149FD}",
-                        "title 1",
-                        "2008-12-15",
-                        new String[] {"subject 1", "second subject"},
-                        "abstract 1",
-                        new String[] {"copyright 1", "copyright 2"},
-                        "dataset",
-                        "Shapefile",
-                        "POLYGON((52.139 5.121, 52.517 5.121, 52.517 4.468, 52.139 4.468, 52.139 5.121))");
-        assertMetacard(mc, expectedValues);
-
-        // verify second metacard's values
-        mc = metacards.get(1);
-        expectedValues = getExpectedMap("{23362852-F370-4369-B0B2-BE74B2859614}",
-                "mc2 title",
-                "2010-12-15",
-                new String[] {"first subject", "subject 2"},
-                "mc2 abstract",
-                new String[] {"first copyright", "second copyright"},
-                "dataset 2",
-                "Shapefile 2",
-                "POLYGON((53.139 6.121, 53.517 6.121, 53.517 5.468, 53.139 5.468, 53.139 6.121))");
-        assertMetacard(mc, expectedValues);
-
-        // verify third metacard's values
-        mc = metacards.get(2);
-        expectedValues = getExpectedMap("{23362852-F370-4369-B0B2-BE74B2859615}",
-                "mc3 title",
-                "2010-12-15",
-                new String[] {"first subject", "subject 3"},
-                "mc3 abstract",
-                new String[] {"first copyright", "second copyright"},
-                "dataset 3",
-                "Shapefile 3",
-                "POLYGON((53.139 6.121, 53.517 6.121, 53.517 5.468, 53.139 5.468, 53.139 6.121))");
-        assertMetacard(mc, expectedValues);
-
+        assertThat(mc.getId(), equalTo("metacard1"));
+        assertThat(mc.getTitle(), equalTo("title1"));
     }
 
     // verifies UTF-8 encoding configured properly when XML includes foreign text with special characters
     @Test
     public void testGetMultipleMetacardsWithForeignText() throws Exception {
+        List<Metacard> inputMetacards = new ArrayList<>();
```

Comment:
They are similar but I don't know if it is worth extracting for the two places we are using it since it is only a few lines.